### ISSUE TITLE
Integration tests fail intermittently

### ISSUE THREAD

**Issue by markspanbroek** [sustainablesource/sustainablesource#38]
Integration tests (on Travis) fail every now and then with the following error:

```
Error: while migrating Migrations: The contract code couldn't be stored, please check your gas limit.
```

**Comment by markspanbroek** [comment]
Hopefully closed in #39. Will reopen should the issue re-occur.

**Comment by markspanbroek** [comment]
Nope, have seen the issue re-occur after #39.

**Comment by markspanbroek** [comment]
A failure log: https://travis-ci.com/sustainablesource/sustainablesource/builds/114062533

**Comment by markspanbroek** [comment]
This is something we could try: https://github.com/trufflesuite/truffle/issues/1308#issuecomment-427457581

**Comment by markspanbroek** [comment]
Hopefully closed with #54. Will re-open should it re-occur.
